boat-scaffold/src/main/templates/boat-java/libraries/resttemplate/ApiClient.mustache

```diff
@@ -42,6 +42,16 @@ import com.fasterxml.jackson.databind.ObjectMapper;
 import org.openapitools.jackson.nullable.JsonNullableModule;
 {{/threetenbp}}
 
+{{#serviceId}}
+import static com.backbase.buildingblocks.backend.communication.http.HttpCommunicationConfiguration.INTER_SERVICE_REST_TEMPLATE_BEAN_NAME;
```

Comment:
I actually use an other rest template in my projects. This is too tied to the Backbase SSDK and interservcie comms. 

@jjjasper @torrespro  Your thoughts?

Reply:
Agreed, this is taken from Transactions I think... Can you point me to your template?

Reply:
removed